sdk/core/azure-core/tests/test_utils.py

```diff
@@ -146,3 +146,43 @@ def test_parse_retry_after():
     assert ret == 0
     ret = parse_retry_after("0.9")
     assert ret == 0.9
+
+
+def test_sanitize_url_query_params():
+    url = "https://www.example.com?q1=value1&q2=value2"
```

Comment:
Still missing a multi-use query parameter test.... E.g. https://www.example.com/foo/bar?q1=hello&q1=smurf....

Reply:
Line 171 technically does this, but I can add an explicit one.